### ISSUE TITLE
Tag npm releases

### ISSUE THREAD

**Issue by mraerino** [Polymer/lit-html#151]
Hi 🙂

I got somewhat confused about the versioning of this library.
The npm version right now is 0.7.1, while the latest tag on github is 0.6.0.

Is there any way we could ensure, that released npm versions are tagged, so that we can figure out from commits what changed. (Changelog would be even better, but since this is not a stable library, maybe not quite there yet.)

I'm under the impression google even has some kind of bot that can do the version release automation for you, isn't there?
Although this could as well just be some npm script.

**Comment by justinfagnani** [comment]
I use `np` on other projects, but it was failing for me here. I'll try to get that working again so releases are tagged.

**Comment by jridgewell** [comment]
Are you using `npm version`? In that case, do `git push --tags`.

**Comment by styfle** [comment]
Here is how I release a quick fix to NPM

```sh
git add -A
git commit -m "Fixed it"
npm version minor
git push --follow-tags
npm publish
```

**Comment by jridgewell** [comment]
`--follow-tags` will only push tags whose related commits are being pushed. `--tags` will push all tags (but only the tags).

**Comment by justinfagnani** [comment]
I've been releasing with `np` which handles this correctly now.

**Comment by mraerino** [comment]
Awesome!

**Comment by jridgewell** [comment]
You're still missing historical version tags. The only tags available right now are `0.6.0` and `0.10.0`.

Please do a `git push origin --tags` (or whatever you call the `Polymer/lit-html` remote repo).
